qa/rpc-tests/fundrawtransaction.py

```diff
@@ -508,13 +508,13 @@ def run_test(self):
         self.sync_all()
 
         for i in range(0,20):
-            self.nodes[0].sendtoaddress(self.nodes[1].getnewaddress(), 0.1)
+            self.nodes[0].sendtoaddress(self.nodes[1].getnewaddress(), 1.1)
         self.nodes[0].generate(1)
         self.sync_all()
 
         #fund a tx with ~20 small inputs
         inputs = []
-        outputs = {self.nodes[0].getnewaddress():0.15,self.nodes[0].getnewaddress():0.04}
+        outputs = {self.nodes[0].getnewaddress():1.15,self.nodes[0].getnewaddress():1.04}
```

Comment:
I'm not sure this was correct, before; it looks like it should be using the outputs from line 511, but wasn't. Probably one for me to check after this PR.